pkg/ingestor/parser/vuln/vuln.go

```diff
@@ -145,13 +154,33 @@ func parseVulns(_ context.Context, s *attestation_vuln.VulnerabilityStatement) (
 			},
 		}
 		ivs = append(ivs, iv)
+
+		var severityErrors error
+		for _, severity := range res.Severity {
+			score, err := parseScoreBasedOnMethod(severity)
+			if err != nil {
+				severityErrors = errors.Join(fmt.Errorf("parsing severity score failed for method %s: %w", severity.Method, err))
+			}
+			vmi = append(vmi, assembler.VulnMetadataIngest{
+				Vulnerability: vuln,
+				VulnMetadata: &generated.VulnerabilityMetadataInputSpec{
+					ScoreType:  generated.VulnerabilityScoreType(severity.Method),
+					ScoreValue: score,
+					Timestamp:  now,
```

Comment:
It seems like OSV has a last modified timestamp, would it make sense to use that?

e.g. in https://google.github.io/osv.dev/get-v1-vulns/
```
  "modified": "2022-04-13T03:04:37.331327Z",
  "published": "2020-06-24T01:51:14.570467Z",
```

Reply:
Seems reasonable.
The question is where to embed this information in the vulnerability attestation since this is the base for the `VulnMetadataIngest` struct.

Reply:
I think this timestamp field works fine, it seems to match up well (to a very close approximation that the score is derived when a vuln entry is updated) with the graphQL spec description (https://github.com/guacsec/guac/blob/main/pkg/assembler/graphql/schema/vulnMetadata.graphql#L62)

Reply:
we chatted during the community all and since this is only for the vulns attestation, so we don't have access to OSV scan info here, so now is fine for now.

Reply:
Actually we had another discussion during the maintainers call on 25 nov, and we were concerned around there being too many nodes generated per timestamp, we landed on having the timestamp be null'ed out, since that information on the scan would be available at the CertifyVuln level.